src/js/components/modal.js

```diff
@@ -0,0 +1,29 @@
+export class cookies {
+    $bodyElement = document.querySelector('body');
+    bodyOverlayClassName = 'overlay';
+    hideClassName = 'hide';
+    showModalClassName = 'show';
+    
+
+    constructor(modalName) {
+        this.modal = document.querySelector(`#${modalName}`);
+        this.closeButton = this.modal.querySelector('button');
+        this.localStorage = localStorage.getItem("cookiesAccepted");
+    }
+
```

Comment:
dodaj sobie 2 metody

setLocalStorage()  //  ona bedzie ustawiac wartosc storage, czyli localStorage.setItem(this.localStorageKey, 'jaka-wartosc');

checkLocalStorage()  //  tutaj ta metoda niech zwraca informacje czy Twoj wpisa w storage wogole istnieje, czyli sprawdzac czy getItem === null